packages/api/app/Utils/Algolia/indexes/service.js

```diff
@@ -13,6 +13,16 @@ const SERVICE_TYPES = {
 const defaultTermMale = 'Não definido';
 const defaultTermFemale = 'Não definida';
 
+/**
+ * Returns an array of keywords
+ *
+ * @param {object} service The service object
+ * @returns {string[]} The service keywords
+ */
+const normalizeKeywords = (service) => {
+	return service.keywords.map((keyword) => keyword.term);
+};
+
```

Comment:
`keywords` belongs to many solutions, what about move it to `core` or somewhere else so we can reuse easily?